modules/sql/src/main/scala/MagnitudeValue.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.sql
+
+import java.math.RoundingMode.HALF_UP
+
+/** Minimal MagnitudeValue class to support reading from the enum
+  * tables.
+  */
+final case class MagnitudeValue(value: BigDecimal) extends Product with Serializable {
+  def toInt: Int = value.underlying.scaleByPowerOfTen(2).setScale(0, HALF_UP).intValue
```

Comment:
Minor nitpick but I might call this `toScaledValue` or something since it is 100x `value`.

Reply:
👍